plugins/presentation/src/components/internal/Properties.svelte

```diff
@@ -55,9 +93,34 @@
       </table>
     </div>
   {/each}
+  <div class="group">
+    {#each mixins as _mixin, i (_mixin._class)}
+      <div class="caption-4" style="cursor:pointer" on:click={(e) => toggleVisible(e)}>
+        {getMixinLabel(_mixin._class)}
+      </div>
+      <table style="display:none">
```

Comment:
Why ```display:none```, it is not clear how it will look, because you modify styles from JS code instead of classes and put definitions here.